medcat/cat.py

```diff
@@ -97,34 +94,36 @@ def __init__(self,
             # Take the new config and assign it to the CDB also
             self.config = config
             self.cdb.config = config
+        self._meta_cats = meta_cats
+        self._create_pipeline(self.config)
 
+    def _create_pipeline(self, config):
```

Comment:
what happens with self.config if you call create_pipeline from a checkpoint

Reply:
`self.config` will be ignored coz some values may not be safe for alteration during training resumption.

Reply:
So the value of cat.config and cat.cdb.config will be different?